app/frontend/src/components/base/BaseDialog.vue

```diff
@@ -23,9 +23,19 @@
         </v-card-text>
       </div>
       <v-card-actions class="justify-center">
-        <v-btn class="BC-Gov-PrimaryButton mb-5" text @click="closeDialog">
-          <slot name="button-text">OK</slot>
-        </v-btn>
+        <div v-if="type == 'OK'">
+          <v-btn class="BC-Gov-PrimaryButton mb-5" text @click="closeDialog">
+            <slot name="button-text">OK</slot>
+          </v-btn>
+        </div>
+        <div v-else-if="type === 'CONTINUE'">
+          <v-btn class="BC-Gov-PrimaryButton light mb-5 mr-5" text @click="closeDialog">
+            <slot name="button-text">Cancel</slot>
+          </v-btn>
+          <v-btn class="BC-Gov-PrimaryButton mb-5" text @click="continueDialog">
+            <slot name="button-text">Continue</slot>
+          </v-btn>
+        </div>
```

Comment:
I'm wondering if we should be making the v-card-actions be more flexible instead and have this entire section be the slot instead of just the button and text. I ran into a similar issue when I was using BaseDialog for the admin panel components. We can allow for more flexibility, but at the potential cost of needing to custom style on the parent level - something we should consider and chat about.

Reply:
Yeah possibly, want to see where we can improve this now that we're using it for more functionality.
Can discuss coming next sprint maybe.